src/App.tsx

```diff
@@ -1,9 +1,15 @@
-import React from 'react';
+import React, { useState } from 'react';
 import './App.scss';
 import { MoviesList } from './components/MoviesList';
 import moviesFromServer from './api/movies.json';
 
 export const App: React.FC = () => {
+  const [query, setValue] = useState('');
+  const visibleMovies = moviesFromServer.filter(movie => (
+    movie.title.toLowerCase().includes(query.toLowerCase())
+    || movie.description.toLowerCase().includes(query.toLowerCase())
```

Comment:
let's set query value in `setQuery` with `.toLowerCase()` so you  wouldn't need to do it twice here
```suggestion
    movie.title.toLowerCase().includes(query)
    || movie.description.toLowerCase().includes(query)
```